Add package readme to System.Numerics.Tensors (#93798)
dotnet/runtime: src/libraries/System.Numerics.Tensors/src/PACKAGE.md

## About

> Provides methods for performing mathematical operations over _tensors_ represented as spans.  These methods are accelerated to use vector operations supported by the CPU where available.

Reviewer: ```suggestion
Provides methods for performing mathematical operations over _tensors_ represented as spans.  These methods are accelerated to use SIMD operations supported by the CPU where available.
```
?

Author: ```suggestion
Provides methods for performing mathematical operations over _tensors_ represented as spans.  These methods are accelerated to use SIMD (Single instruction, multiple data) operations supported by the CPU where available.
```

## Key Features

> * Numerical operations on Vectors represented as `ReadOnlySpan<float>`

Author: ```suggestion
* Numerical operations on tensors represented as `ReadOnlySpan<float>`
```

> * Numerical operations on Vectors represented as `ReadOnlySpan<float>`

Reviewer: ```suggestion
* Numerical operations on tensors represented as `ReadOnlySpan<float>`
```

* Element-wise arithmetic: Add, Subtract, Multiply, Divide, Exp, Log, Cosh, Tanh, etc.

> * Vector arithmetic: CosineSimilarity, Distance, Dot, Normalize, Softmax, Sigmoid, etc.

Reviewer: ```suggestion
* Tensor arithmetic: CosineSimilarity, Distance, Dot, Normalize, Softmax, Sigmoid, etc.
```
